feat: Added app icon assets

dilumdesilva opened the pull request:
Fixes #19 

Changes: Added corresponding app icon assets as of apple app icon [guideliness](https://developer.apple.com/design/human-interface-guidelines/ios/icons-and-images/app-icon/)

Screenshots for the change: 

<img width="384" alt="Screen Shot 2019-03-30 at 12 25 57 AM" src="https://user-images.githubusercontent.com/23298579/55256455-25ce9600-5283-11e9-807e-50b6b9ef6550.png">

dilumdesilva committed:
feat: Added app icon assets

Added corresponding app icon assets as of apple app icon guideliness

Implement #19

dilumdesilva committed:
feat: Added app icon assets

Added corresponding app icon assets as of apple app icon guideliness

Implement #19

fix: Removed the black shadow in app icon

fixes: #19 Resolve !22#pullrequestreview-221243491

chashmeetsingh reviewed (changes requested):
There seems to be a black background visible meaning the icon size is smaller. Please correct that.

dilum1995 commented:
@chashmeetsingh oh yeah I also noticed that but I double checked the sizes of the icons and there are no any issues with the sizes of the icons. I think we see a black background because of the icon shape. I will work on that and submit it back

dilumdesilva committed:
fix: Removed the black shadow in app icon

fixes: #19 Resolve !22#pullrequestreview-221243491

dilum1995 commented:
@chashmeetsingh I have already pushed the fix and I squashed the commits. Now the app icon appears correctly.

![Screen Shot 2019-04-06 at 5 30 26 AM](https://user-images.githubusercontent.com/23298579/55661996-5d15e780-582d-11e9-9c11-33a3c1c9aa0f.png)

dilumdesilva committed:
new changes

dilumdesilva committed:
feat: Getting started screen with user account validation via email

Implemented the getting started screen which asks ueser for the email to validate whether the
user already has an account or not.
if the user has an account already the starting screen will navigate directly to the login screen
after the email validation. if user doesn't have an account it will redirect user to the signup screen.

Implemented #20

dilumdesilva committed:
Merge pull request #1 from dilum1995/feature/#20-emailValidation

Feature/#20 email validation
